docs/release-notes/2.35.0.md

```diff
@@ -0,0 +1,138 @@
+# Percona Monitoring and Management 2.35.0
+
+
+| **Release date:** | [Feb 21, 2023]                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+
+## Release Highlights
+
+### Access Control: Limit access to metrics
+
+!!! caution alert alert-warning "Important/Caution"
+    Disclaimer:  PMM Access Control is still in Technical Preview and is subject to change. We recommend that early adopters use this feature for testing purposes only.
+
+We are pleased to announce that PMM 2.35.0 marks the first step towards introducing PMM Access Control. Label-based access control in PMM allows you to manage who has access to metrics based on labels. By creating roles, you can specify which data can be queried based on specific label criteria, for instance, allowing the QA team to view data related to test environments. Using this feature, you can associate multiple labels with a role, ensuring only data from series that match your defined labels is returned. This feature provides a standardized method for granting, changing, and revoking access to metrics, giving you a powerful and flexible way to manage access to metrics and ensure secure access to the information you need. For more information, see Access Control documentation.
+
+### GA: Deployment of PMM in Kubernetes with Helm Chart
+
+We are happy to announce the General Availability of PMM deployment in Kubernetes with the Helm chart. PMM deployment via Helm chart in Kubernetes has been available as Tech Preview since PMM v2.29, but now we're delighted to offer it to users as a fully supported feature. For more information, see [documentation](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/helm.html).  
+
+### DBaaS: Backup schedule while creating DB clusters
+
+Starting with PMM 2.35.0, you can add a backup schedule while creating DB clusters in DBaaS. This feature is a fusion of backup management and DBaaS in PMM. Currently, DBaaS only supports scheduled backups, which can only be enabled when a database cluster is created.
+
+!!! note alert alert-primary "Note"
+    DBaaS does not support on-demand backups.
+
+### DBaaS: Restore cluster from backup stored on S3
+
+You can now create a DBaaS cluster from a backup stored on S3. Currently, you cannot restore a cluster from a backup. You can only create a DBaaS database cluster from backup.
```

Comment:
Sharing some feedback from @nethalo (slightly rephrased):
```suggestion
You can now create a DBaaS cluster from a backup stored on S3. Backups from an existing cluster can only be used to create a NEW DBaaS cluster. Attempts to use it to restore a previous version of the cluster will result in downtime since the system will treat it as a creation of a new cluster.
`